src/components/HomePage/HothDescription.js

```diff
@@ -109,6 +116,23 @@ export default function HothDescription() {
 						}}/>
 					</Grid>
 				</Grid>
+				<Grid container spacing={isSmall ? 3 : 8} alignItems='center' justify='center'>
+					<Grid item xs={12} sm={8} md={6}>
```

Comment:
We're repeating a lot of these Grid components. Since there are more than 2 instances of these let's move it out into another component just to make it more readable. 

You can do something like `RightGridItem` and `LeftGridItem` for the right and left aligned components. 

So something like 
```jsx
const LeftGridItem = <Grid item xs={12} sm={8} md={6} />;
```

This code should automatically take in `children` (aka children components) so you'll be able to use it just as we did before. It will be used something like this:
```jsx
<LeftGridItem>
    <Typography>bloop</Typography>
</LeftGridItem>
```